src/Libraries/RevitNodes/Elements/TextNote.cs

```diff
@@ -233,14 +233,6 @@ public string Text
         }
 
         /// <summary>
-        /// Get geometrical curves from the text this TextNote contains.
-        /// </summary>
-        public new Autodesk.DesignScript.Geometry.Curve[] Curves
-        {
-            get { return base.Curves(); }
-        }
-
```

Comment:
@mjkkirschner Was this added in 1.3 or 2.0?  I assume since this was recently added it wasn't being used extensively.  Just wondering in terms of producing dummy nodes in existing graphs.  Otherwise LGTM

Reply:
@alfarok @mjkkirschner Correct me if wrong. I think this change was only made to 2.0 so would only affect daily build users.